From cc0ccb75be38b14c2941987899c20c54ba12e846 Mon Sep 17 00:00:00 2001
From: Chris Bieneman <beanz@apple.com>
Date: Thu, 28 Apr 2016 21:16:45 +0000
Subject: [PATCH 2010/2012] [CMake] Adding another missing include. NFC.

This also works fine today, but will break with my upcoming refactoring.

git-svn-id: https://llvm.org/svn/llvm-project/compiler-rt/trunk@267941 91177308-0d34-0410-b5e6-96231b3b80d8
(cherry picked from commit 9e256d0505936e31fb742ce73b39ae9d53cd2fe8)
---
 cmake/Modules/CompilerRTDarwinUtils.cmake | 2 ++
 1 file changed, 2 insertions(+)

diff --git llvm_release_38/projects/compiler-rt/cmake/Modules/CompilerRTDarwinUtils.cmake macports_release_38/projects/compiler-rt/cmake/Modules/CompilerRTDarwinUtils.cmake
index 2902f9d..e8f0fe6 100644
--- llvm_release_38/projects/compiler-rt/cmake/Modules/CompilerRTDarwinUtils.cmake
+++ macports_release_38/projects/compiler-rt/cmake/Modules/CompilerRTDarwinUtils.cmake
@@ -1,3 +1,5 @@
+include(CMakeParseArguments)
+
 # On OS X SDKs can be installed anywhere on the base system and xcode-select can
 # set the default Xcode to use. This function finds the SDKs that are present in
 # the current Xcode.
-- 
2.10.1

